gems/pending/util/miq-process.rb

```diff
@@ -1,6 +1,7 @@
 # encoding: US-ASCII
 
 require 'sys-uname'
+require 'sys/proctable'
```

Comment:
@Fryguy changed this to use `sys/proctable` instead of `sys-proctable` for consistency in this PR.